monkey/monkey_island/cc/repository/i_log_repository.py

```diff
@@ -2,8 +2,12 @@
 from typing import Optional, Sequence
 
 
+# TODO: Actually define the Log class
+class Log:
+    pass
+
+
 class ILogRepository(ABC):
```

Comment:
We can remove this file too. The log contents will be stored on the `Agent` object.